CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # salsify_rubocop
 
+## 1.59.1
```

Comment:
❓ Shouldn't this be a major bump as it is making `salsify_rubocop` more strict (by adding more rules) and hence making it a non-backwards compatible change (ex: with `1.59.0` rubocop passed but with `1.59.1` it might not because of the `rubocop-rake` checks)?